plugin/output/splunk/splunk.go

```diff
@@ -98,6 +100,14 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.OutputPluginP
 	p.config = config.(*Config)
 	p.client = p.newClient(p.config.RequestTimeout_)
 
+	ctx := context.TODO()
+	stdBackoff := backoff.NewExponentialBackOff()
+	stdBackoff.Multiplier = 1.2
+	stdBackoff.RandomizationFactor = 0.25
+	stdBackoff.InitialInterval = time.Second
+	stdBackoff.MaxInterval = stdBackoff.InitialInterval * 2
+	p.backoff = backoff.WithContext(stdBackoff, ctx)
+
```

Comment:
same here, lots of magic constants. let's use a function for this

Reply:
Same as https://github.com/ozontech/file.d/pull/74#discussion_r859654027